docs/sources/features/explore/index.md

```diff
@@ -54,7 +54,8 @@ You can close the newly created query by clicking on the Close Split button.
 ## Query history
 
 > BETA: Query history is a beta feature.
-> Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click the "Query history" button in Explore.
+
+Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click the "Query history" button in Explore.
```

Comment:
```suggestion
Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click **Query history** in Explore.
```

Reply:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md) and [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/), UI items should be bold format.